server/lib/classes/http-incoming.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+
+/**
+ * A bearer object to hold misc data through a http
+ * request / response cyclus.
+ * @class HttpIncoming
+ */
+const HttpIncoming = class HttpIncoming {
+    constructor(request, {
+        version = '',
+        extras = '',
+        name = '',
+        type = '',
+        org = '',
+    } = {}) {
+        this._version = version;
```

Comment:
side discussion, are we leaning towards underscores or symbols for private props these days?

Reply:
Will be symbols. Just did Hungarian notation for easiness during initial development.